django/contrib/admin/utils.py

```diff
@@ -296,10 +296,14 @@ def lookup_field(name, obj, model_admin=None):
             attr = getattr(model_admin, name)
             value = attr(obj)
         else:
-            attr = getattr(obj, name)
+            attr = getattr(obj, name, None)
```

Comment:
I'm thinking that we may need to use some sort of sentinel to be able to discern whether the attribute `name` is actually `None` or it really does not exist.